app/src/main/res/layout/activity_passenger_list.xml

```diff
@@ -6,4 +6,136 @@
     android:layout_height="match_parent"
     tools:context=".ui.passenger.PassengerListActivity">
 
+    <TextView
+        android:id="@+id/logo"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="BUSIFY"
+        android:paddingStart="25dp"
+        android:textSize="48sp"
+        android:textStyle="bold"
+        android:background="#FBBC05"
+        android:textColor="@color/white"
+        />
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/search"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical"
+        app:layout_constraintTop_toBottomOf="@+id/logo"
+        app:cardCornerRadius="50dp"
+        android:layout_marginStart="25dp"
+        android:layout_marginEnd="25dp"
+        android:layout_marginTop="18dp"
+        >
+        <androidx.appcompat.widget.LinearLayoutCompat
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center_vertical"
+            android:padding="10dp"
+            >
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:src="@drawable/search_icon"
```

Comment:
You can use a search view instead. Refer [this](https://developer.android.com/reference/android/widget/SearchView).